File: dto-mapping-benchmark/src/main/java/com/example/dtomappingbenchmark/v2/entity/Post.java
package com.example.dtomappingbenchmark.v2.entity;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
@Entity(name = "PostV2")
public class Post {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    Long id;

    @Column
    private String title;
    @Column
    private String content;

    @ManyToOne(fetch = FetchType.LAZY)
    private User user;
}

File: object-mapping-benchmark-v1/src/main/kotlin/yu/selab/cj/objectmappingbenchmarkv1/post/entity/Post.kt
package yu.selab.cj.objectmappingbenchmarkv1.post.entity

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.JoinColumn
import jakarta.persistence.ManyToOne
import yu.selab.cj.objectmappingbenchmarkv1.user.entity.User

@Entity
class Post(

    @Column
    val title : String,

    @Column
    val content : String?,

    @ManyToOne
    @JoinColumn(name = "uid")
    val user : User
) {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val pid : Long = 0

}